<launch>
	<!-- this makes it work on either the VM or racecar without arguments -->
	<remap from="/odom" to="/vesc/odom"/>

	<arg name="scan_topic" default="/scan"/>
	<arg name="odometry_topic" default="/odom"/>
	<arg name="max_particles" default="4000"/>
	
	<node pkg="final_challenge" type="particle_filter.py" name="Particle_filter" output="screen">
		<param name="scan_topic" value="$(arg scan_topic)"/>
		<param name="odometry_topic" value="$(arg odometry_topic)"/>
		<param name="max_particles" value="$(arg max_particles)"/>
		<param name="max_viz_particles" value="100"/>
		<param name="range_method" value="cddt"/>

		<param name="theta_discretization" value="108"/>
		<!-- max sensor range in meters -->
		<param name="max_range" value="10"/> 
	</node>
</launch>
